release-tools/travis.yml

```diff
@@ -14,6 +14,7 @@ before_script:
 - export PATH=$PWD/bin:$PATH
 script:
 - make -k all test GOFLAGS_VENDOR=$( [ -d vendor ] && echo '-mod=vendor' )
+- make e2e-tests
```

Comment:
does this work in your env?

Reply:
some of the tests fails.
```                                                                                                                                                                                                      
Summarizing 2 Failures:                                                                                                                                                                                            
                                                                                                                                                                                                                   
[Fail] [sig-storage] CSI Volumes [Driver: nfs.csi.k8s.io] [Testpattern: Pre-provisioned PV (default fs)] subPath [It] should unmount if pod is gracefully deleted while kubelet is down [Disruptive][Slow]         
/home/manohar/go/pkg/mod/k8s.io/kubernetes@v1.14.1/test/e2e/storage/utils/utils.go:206                                                                                                                             
                                                                                                                                                                                                                   
[Fail] [sig-storage] CSI Volumes [Driver: nfs.csi.k8s.io] [Testpattern: Pre-provisioned PV (default fs)] subPath [It] should unmount if pod is force deleted while kubelet is down [Disruptive][Slow]              
/home/manohar/go/pkg/mod/k8s.io/kubernetes@v1.14.1/test/e2e/storage/utils/utils.go:206                                                                                                                             
                                                                                                                                                                                                                   
Ran 20 of 83 Specs in 1293.726 seconds                                                                                                                                                                             
FAIL! -- 18 Passed | 2 Failed | 0 Pending | 63 Skipped
```
I didn't run the tests to completion. I stopped after 20 mins.

Reply:
I think we can skip the Disruptive tests. They try to ssh into the node where the kubelet is running, but that functionality is not available in all environments.

Reply:
How do I skip those tests? I might have to the run through all the 83 specs and skip the failing tests?

Reply:
When you invoke ginkgo you can specify a "skip" flag with the regex of the test cases to skip